addons/account/models/account_journal.py

```diff
@@ -357,7 +357,11 @@ def _onchange_type(self):
 
     def _get_alias_values(self, type, alias_name=None):
         if not alias_name:
-            alias_name = self.name
+            try:
+                remove_accents(alias_name).encode('ascii')
```

Comment:
Hummm... did you mean ...?
```py
   alias_name = remove_accents(self.name).encode('ascii')
```
because bool(alias_name) == False, as per line 359, and doing this `remove_accents` isn't doing much

Reply:
My bad, I meant something like this